### ISSUE TITLE
copycds doesn't recognize CentOS 8.x ISO images

### ISSUE THREAD

**Issue by conxuro** [xcat2/xcat-core#6576]
Trying to generate an image from Centos 8.0 and/or CentOS 8.1 ISO results in error.

```
$ copycds CentOS-8-x86_64-1905-dvd1.iso 
Error: [hpchead]: copycds could not identify the ISO supplied, you may wish to try -n <osver>

$ copycds CentOS-8.1.1911-x86_64-dvd1.iso 
Error: [hpchead]: copycds could not identify the ISO supplied, you may wish to try -n <osver>
```

It works with -n centos8.0 (or centos8.1) -a x86_64 but doesn't recognize ISO automatically like with CentOS 7.x ISO images.

---

Software used:
```
$ cat /etc/redhat-release 
CentOS Linux release 7.7.1908 (Core)

$ xcatconfig -v
Version 2.15 (git commit 218c6d3acc8bdbd7f72115e48cda2b1a3613d18a, built Mon Nov  4 15:17:59 EST 2019)
```

**Comment by gurevichmark** [comment]
@conxuro 
CentOS 8.0 appears to be working with the latest development version of xCAT

```
[root@stratton01 ~]# lsxcatd -a
Version 2.15.1 (git commit 4dd25923817d6f4303fbddab3344f7d118815c98, built Thu Feb 20 00:34:44 EST 2020)
This is a Management Node
dbengine=SQLite
```
```
[root@stratton01 ~]# copycds /mnt/xcat/iso/centos/8.0/CentOS-8-x86_64-1905-dvd1.iso
Copying media to /install/centos8/x86_64
Media copy operation successful

[root@stratton01 ~]# lsdef -t osimage centos8-x86_64-install-compute
Object name: centos8-x86_64-install-compute
    imagetype=linux
    osarch=x86_64
    osdistroname=centos8-x86_64
    osname=Linux
    osvers=centos8
    otherpkgdir=/install/post/otherpkgs/centos8/x86_64
    pkgdir=/install/centos8/x86_64
    pkglist=/opt/xcat/share/xcat/install/centos/compute.centos8.pkglist
    profile=compute
    provmethod=install
    template=/opt/xcat/share/xcat/install/centos/compute.centos8.tmpl
[root@stratton01 ~]#
```

But for CentOS 8.1, `copycds` recognizes the osname and arch, but creates the osimage definition with the same name as CentOS8.0
```
[root@stratton01 ~]# copycds /mnt/xcat/iso/centos/8.1/CentOS-8.1.1911-x86_64-dvd1.iso
Copying media to /install/centos8/x86_64
Media copy operation successful

[root@stratton01 ~]# lsdef -t osimage centos8-x86_64-install-compute
Object name: centos8-x86_64-install-compute
    imagetype=linux
    osarch=x86_64
    osdistroname=centos8-x86_64
    osname=Linux
    osvers=centos8
    otherpkgdir=/install/post/otherpkgs/centos8/x86_64
    pkgdir=/install/centos8/x86_64
    pkglist=/opt/xcat/share/xcat/install/centos/compute.centos8.pkglist
    profile=compute
    provmethod=install
    template=/opt/xcat/share/xcat/install/centos/compute.centos8.tmpl

[root@stratton01 ~]# ls /install/centos8/x86_64/BaseOS/Packages/centos-repos-8.1-1.1911.0.8.el8.x86_64.rpm
/install/centos8/x86_64/BaseOS/Packages/centos-repos-8.1-1.1911.0.8.el8.x86_64.rpm
[root@stratton01 ~]#
```

**Comment by gurevichmark** [comment]
The reason appears to be that `.diskinfo` file in the 8.1 ISO lists it as `CentOS Linux 8`
```
[root@stratton01 xcat]# mount -o loop /mnt/xcat/iso/centos/8.1/CentOS-8.1.1911-x86_64-dvd1.iso /tmp/gurevich/centos_iso

[root@stratton01 xcat]# cat /tmp/gurevich/centos_iso/.discinfo
1578087695.538488
CentOS Linux 8
x86_64
ALL
[root@stratton01 xcat]#
```

**Comment by gurevichmark** [comment]
Fixed by #6581
